validator/keymanager/remote-web3signer/keymanager.go

```diff
@@ -36,34 +38,36 @@ type SetupConfig struct {
 
 // Keymanager defines the web3signer keymanager.
 type Keymanager struct {
-	client                httpSignerClient
+	client                internal.HttpSignerClient
 	genesisValidatorsRoot []byte
 	publicKeysURL         string
 	providedPublicKeys    [][48]byte
 	accountsChangedFeed   *event.Feed
+	validator             *validator.Validate
 }
 
 // NewKeymanager instantiates a new web3signer key manager.
 func NewKeymanager(_ context.Context, cfg *SetupConfig) (*Keymanager, error) {
-	if cfg.BaseEndpoint == "" || len(cfg.GenesisValidatorsRoot) == 0 {
-		return nil, fmt.Errorf("invalid setup config, one or more configs are empty: BaseEndpoint: %v, GenesisValidatorsRoot: %v", cfg.BaseEndpoint, cfg.GenesisValidatorsRoot)
+	if cfg.BaseEndpoint == "" || len(cfg.GenesisValidatorsRoot) != fieldparams.RootLength || bytes.Equal(params.BeaconConfig().ZeroHash[:], cfg.GenesisValidatorsRoot) {
```

Comment:
We perform this validation in a few places. If we only do this in this package, maybe we can turn this validation into a helper function that is unexported. If we use it in several packages, we can find a better place to put that helper function

Reply:
create new function in bytesutil